payments_service/auth/views.py

```diff
@@ -45,6 +45,12 @@ def post(self, request):
         request.session['buyer_pk'] = buyer['resource_pk']
         request.session['buyer_uuid'] = buyer['uuid']
 
+        # As a convenience, put any saved payment methods in the response
+        # if the user has them.
+        pay_methods = api.braintree.mozilla.paymethod.get(
+            active=1, braintree_buyer__buyer__uuid=buyer['uuid'])
```

Comment:
heh, I was scared to try. Let me see if it works. Hopefully slumber does the right thing.